### ISSUE TITLE
Convergence issue when training a model

### ISSUE THREAD

**Issue by hoseong-kim** [pedro-morgado/score-zeroshot#2]
When I tried to train GoogLeNet (fine-tuning) on CUB dataset with attributes, it did not converge. I just followed hyperparameters in SCoRe paper and this github code. The hyperparameters are as follows. 
python score_train.py G_Attr CUB Attributes GoogLeNet -g 0.01 -c 1.0 --iters 700 --gpu 0 --init_lr 0.0005 --snapshot ./G_Attr/inception_v1.caffemodel |& tee ./G_Attr/train_G_Attr.log

test_iter: 20
test_iter: 20
test_interval: 100
base_lr: 0.0005
display: 20
max_iter: 700
lr_policy: "fixed"
momentum: 0.9
weight_decay: 0.0005
snapshot: 5000

I trained on Ubuntu 16.04 with GTX 1080 Ti, CUDA 8.0, and cuDNN v7.
Could you give me any hints for convergence?

Results: 
![image](https://user-images.githubusercontent.com/25577094/34289152-6e1a269c-e734-11e7-8183-f3e1d4370f33.png)
I1222 15:54:06.236789 26844 solver.cpp:397]     Test net output #0: SCoRe/eval/obj/accuracy = 0.0210938
I1222 15:54:06.388346 26844 solver.cpp:218] Iteration 700 (3.90624 iter/s, 5.12002s/20 iters), loss = 5.02677
I1222 15:54:06.394364 26844 solver.cpp:237]     Train net output #0: SCoRe/cwReg = 0.0053981 (* 1 = 0.0053981 loss)
I1222 15:54:06.394374 26844 solver.cpp:237]     Train net output 1: SCoRe/eval/obj/accuracy = 0
I1222 15:54:06.394381 26844 solver.cpp:237]     Train net output 2: SCoRe/objLoss = 5.02252 (* 0.990099 = 4.9728 loss)
I1222 15:54:06.394387 26844 solver.cpp:237]     Train net output #3: SCoRe/semLoss = 216.479 (* 0.000224393 = 0.0485763 loss)
I1222 15:54:06.394403 26844 sgd_solver.cpp:105] Iteration 700, lr = 0.0005

**Comment by hoseong-kim** [comment]
In score_model.py, "ns[name] = L.Accuracy(\*[xObj, xLb], name=name)" is replaced by
"ns[name] = L.Accuracy(\*[xObj, xLb], name=name, include={'phase': caffe.TEST})".

Now I fixed it by adding "include { phase: TEST }" of "Accuracy" layer in "train.prototxt".
Calculating accuracy when training might cause the error (=not converging).

You can also see #3.
